presto-native-execution/presto_cpp/main/common/Utils.h

```diff
@@ -31,4 +31,9 @@ std::shared_ptr<folly::SSLContext> createSSLContext(
     const std::string& clientCertAndKeyPath,
     const std::string& ciphers);
 
+/// Returns process-wide CPU time in nanoseconds.
```

Comment:
It would be nice to avoid files / classes named Util as they tend to become a "dumping ground" for lots of unrelated code. It would be better to come with more specific names.